deps/lua/overlay.BUILD.bazel

```diff
@@ -0,0 +1,29 @@
+load("@rules_license//rules:license.bzl", "license")
+load("@rules_cc//cc:cc_library.bzl", "cc_library")
+load("@rules_cc//cc:cc_shared_library.bzl", "cc_shared_library")
+
+package(default_package_metadata = [":license"])
+
+license(
+    name = "license",
+    license_kinds = ["@rules_license//licenses/spdx:MIT"],
```

Comment:
@rules_license//licenses/generic:unencumbered"

Reply:
https://lua.org/license.html says `Lua is free software distributed under the terms of the [MIT license](https://opensource.org/license/mit/)` so I'm not sure "unencumbered" is correct here